config.yaml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
(not related to this file)
We may want to modify the readme, I'm not sure the current description is correct.

Reply:
I updated the readme